rocAL/include/file_io_stream.h

```diff
@@ -0,0 +1,22 @@
+
```

Comment:
Copyright missing. Why do we have a separate file for this class? If it's used only in webdataset reader, please add to appropriate file.

Reply:
File has been removed